tasks/main.yml

```diff
@@ -125,13 +125,34 @@
   when:
     - deploydrupal_site_install
 
+# Config Value Check - This is primarily to catch configuration value updates
+# done via update hooks from Dependabot PR's. This is to replace the dev process
+# that would be done in a local environment.
+- name: config value check
+  shell: |
+    "{{ deploydrupal_drush_path }} updatedb -y"
+    "{{ deploydrupal_drush_path }} config:export -y | grep 'identical'"
```

Comment:
Should we be more specific and check for a longer string? It may be far-fetched, but what if there is a config file with the word `identical` in the path name somewhere?

Reply:
If we want to be stricter about it, we might want to use something built for change-detection. We're working in the context of a `git` repo, right? In that case, this works as well:

`{{ deploydrupal_drush_path }} config:export -y ; git status ./config | ag 'nothing to commit'`

Reply:
I like that.

Reply:
Added a longer string. 726c6f47f3a78ed7c0a726065d0d2064f32d1d48

Choosing not to implement the `git status` idea per our discussion where we realized this could be an issue if we deploy code to an environment without `git`. Also, the `grep` solution should work just fine.